drivers/hmis/lib/form_data/springfield/fragments/patches/4_11_dv_rule.json

```diff
@@ -0,0 +1,323 @@
+          "variable": "projectFunderComponents",
+          "operator": "INCLUDE",
+          "value": "HUD: HUD-VASH"
+        },
+        {
+          "operator": "ALL",
+          "_comment": "HUD: PFS – Collection required for all permanent housing projects",
+          "parts": [
+            {
+              "variable": "projectFunderComponents",
+              "operator": "INCLUDE",
+              "value": "HUD: PFS"
+            },
+            {
+              "operator": "ANY",
+              "parts": [
+                {
+                  "variable": "projectType",
+                  "operator": "EQUAL",
+                  "value": 3
+                },
+                {
+                  "variable": "projectType",
+                  "operator": "EQUAL",
+                  "value": 9
+                },
+                {
+                  "variable": "projectType",
+                  "operator": "EQUAL",
+                  "value": 10
+                },
+                {
+                  "variable": "projectType",
+                  "operator": "EQUAL",
+                  "value": 13
+                }
+              ]
+            }
+          ]
+        },
+        {
+          "_comment": "HHS: PATH – Collection required for all components",
+          "variable": "projectFunderComponents",
+          "operator": "INCLUDE",
+          "value": "HHS: PATH"
+        },
+        {
+          "operator": "ALL",
+          "_comment": "VA: SSVF – Collection required for RRH and Homelessness Prevention",
+          "parts": [
+            {
+              "variable": "projectFunderComponents",
+              "operator": "INCLUDE",
+              "value": "VA: SSVF"
+            },
+            {
+              "operator": "ANY",
+              "parts": [
+                {
+                  "variable": "projectType",
+                  "operator": "EQUAL",
+                  "value": 12
+                },
+                {
+                  "variable": "projectType",
+                  "operator": "EQUAL",
+                  "value": 13
+                }
+              ]
+            }
+          ]
+        },
+        {
+          "_comment": "VA: GPD – Collection required for all components",
+          "variable": "projectFunderComponents",
+          "operator": "INCLUDE",
+          "value": "VA: GPD"
+        },
+        {
+          "_comment": "VA: Community Contract Safe Haven",
+          "variable": "projectFunders",
+          "operator": "INCLUDE",
+          "value": 30
+        },
+        {
+          "variable": "projectFunderComponents",
+          "operator": "INCLUDE",
+          "value": "VA: CRS Contract Residential Services"
+        },
+        {
+          "_comment": "YHDP",
+          "variable": "projectFunders",
+          "operator": "INCLUDE",
+          "value": 43
+        },
+        {
+          "variable": "projectId",
+          "operator": "EQUAL",
+          "value": "762"
+        },
```

Comment:
This seems like it's super brittle, not sure what else we can do if we need to  target specific projects, but I guess I'd rather see an attribute added to the project table and then look for that.

Reply:
I'm not sure how an attribute on the project able would work. There isn't something specific linking these projects together except for the fact that they should ask these questions (afaik). I guess we could do a custom data element like "asks_psde_questions_on_assessments" and attach it to the Project, and then modify the form `rule` logic to support specifying something like ` { "variable": "customDataElement", "key": "asks_psde_questions_on_assessments", "operator": "EQUAL", "value": true }`. Or we could use a Project Group and make this `projectGroupId`.

Also though, it doesn't seem crazy to me that in the form builder, you can just specify a specific project that the question is enabled for. And that it  needs to be updated if there is a new project that should have the question enabled.

Reply:
We do this elsewhere to specify a specific set of projects that allow multi-unit households. I think a Project Group is probably the most appropriate way to do this, but we don't have those yet.

Reply:
Adding a custom field to project was what I was thinking, or potentially a project group, but that has similar issues in that we're either making a system project group where you can't change the name, or looking for a DB ID.  I think it being the projectID is probably ok, even a multi-installation HMIS will need unique projectIds within the HMIS.

Reply:
"custom field to project" meaning a CustomDataElement though, right?